### PR TITLE
Develop eric

### PR DESCRIPTION
**Pull request by ERICCYS**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,7 +33,8 @@ bower_components
 build/Release
 
 # Dependency directories
-node_modules/
+client/node_modules/
+sever/node_modules/
```
Comment (MaxPoon) [inline .gitignore:37]: I guess just `node_modules/` is enough?

**Review by MaxPoon — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (ERICCYS) [reply .gitignore:37]: I think the client side and the server side got their own different dependencies? So I think the dependency installing will be conducted inside two directories.

Comment (MaxPoon) [reply .gitignore:37]: Yes, but I believe adding `node_modules/` in `.gitignore` means ignoring all directories called `node_modules` in current folder and any subfolders. So both `client/node_modules/` and `sever/node_modules/` will be ignored.

Comment (ERICCYS) [reply .gitignore:37]: Oh, I see. Learnt this from here and will change it back. Thank you Max!

**Pull request merged**